modules/exoplanets/exoplanetsmodule.h

```diff
@@ -38,11 +40,30 @@ class ExoplanetsModule : public OpenSpaceModule {
     ExoplanetsModule();
     virtual ~ExoplanetsModule() = default;
 
+    const std::string exoplanetsDataPath() const;
```

Comment:
No need to impose the `const`ness on the caller, since it doesn't do anything as a copy is returned anyway and that copy doesn't need to be `const`.  It should be up to the caller to decide whether they want to store it as `const` variable or not;
```
  std::vector a = module->exoplanetsDataPath();
  const std::vector b = module->exoplanetsDataPath();
```
both compile and do the same thing